flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarOperatorsTest.scala

```diff
@@ -304,6 +422,67 @@ class ScalarOperatorsTest extends ScalarOperatorsTestBase {
       "SHIFTRIGHTUNSIGNED(-64,-1)",
       "1"
     )
+
+    //special shift test
+    val b1 = Byte.MinValue
+    testAllApis(
+      b1.cast(Types.BYTE).shiftRightUnsigned(9),
+      s"$b1.cast(BYTE).shiftRightUnsigned(9)",
+      s"SHIFTRIGHTUNSIGNED(CAST($b1 AS TINYINT), 9)",
+      "-1"
+    )
+
+    testAllApis(
+      b1.cast(Types.BYTE).shiftRightUnsigned(17),
+      s"$b1.cast(BYTE).shiftRightUnsigned(17)",
+      s"SHIFTRIGHTUNSIGNED(CAST($b1 AS TINYINT), 17)",
```

Comment:
ditto, replace `17` with `33` for tiny int.

Reply:
Thanks for review~ @pnowojski , I updated the func name and tests